randovania/gui/dialog/game_input_dialog.py

```diff
@@ -36,16 +36,25 @@ def __init__(self, options: Options, patcher: Patcher, word_hash: str, spoiler:
 
         per_game = options.options_for_game(self._game)
 
-        description_text = "<html><head/><body><p>In order to create the randomized game, an ISO file of {} for the Nintendo Gamecube is necessary.</p>".format(
-            game.long_name)
+        file_format1, platform, file_format2 = game.data.gui().input_file_text
+        description_text = "<html><head/><body><p>In order to create the randomized game, {} of {} for {} is necessary.</p>".format(
+            file_format1, game.long_name, platform)
         if not patcher.uses_input_file_directly:
             description_text += "<p>After using it once, a copy is kept by Randovania for later use.</p>"
         description_text += "</body></html>"
         self.description_label.setText(description_text)
 
         # Input
-        self.input_file_edit.textChanged.connect(self._validate_input_file)
-        self.input_file_button.clicked.connect(self._on_input_file_button)
+        if patcher.requires_input_file:
+            self.input_file_label.setText(f"Input File (Vanilla {file_format2})")
+            self.input_file_edit.setPlaceholderText(f"Path to vanilla {file_format2}")
+            self.input_file_edit.textChanged.connect(self._validate_input_file)
+            self.input_file_button.clicked.connect(self._on_input_file_button)
+        else:
+            self.input_file_label.setVisible(False)
```

Comment:
I think the interface would be more intuitive if it would state that "Game does not require any input files."

Reply:
:+1: